homeassistant/components/modbus/__init__.py

```diff
@@ -71,9 +66,8 @@
     DEFAULT_STRUCTURE_PREFIX,
     DEFAULT_TEMP_UNIT,
     MODBUS_DOMAIN as DOMAIN,
-    SERVICE_WRITE_COIL,
-    SERVICE_WRITE_REGISTER,
 )
+from .modbus import modbus_setup
```

Comment:
Is this change intended for this PR? We should preferably only refactor the tests and not the code under test at the same time if possible.